tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -464,6 +464,11 @@ Then('the file {string} should be in {string} state on the webUI after a page re
   await client.refresh()
   return client.page.sharedWithMePage().assertDesiredStatusIsPresent(filename, status)
 })
+
 Then('the autocomplete list should not be displayed on the webUI', function () {
   return client.page.FilesPageElement.sharingDialog().assertAutocompleteListIsNotVisible()
 })
+
+When('user {string} shares file/folder {string} with group {string}', function (sharer, elementToShare, receiver) {
```

Comment:
The order in this context is not perfect, but it'd be better if they were. I'd suggest moving this to the group where `When` statements are clustered.